File: www/cgi-bin/php/test_status.php
#!/usr/bin/env php-cgi
<?php
// Return custom HTTP status
$message = "I'm a teapot";

header("Status: 418 $message");
header("Content-Type: text/html");

echo "<html><body><h1>418 $message</h1>";
echo "<p>This CGI returned a custom status code.</p>";
echo "</body></html>";
?>

File: www/cgi-bin/python/test_status.py
#!/usr/bin/env python3

# CGI output with custom status (404)
body = "<html><body><h1>404 Custom Error</h1><p>This CGI returns a 404 status.</p></body></html>"

print("Status: 404 Not Found")
print("Content-Type: text/html")
print()
print(body)
